plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py

```diff
@@ -4,81 +4,49 @@
 import shutil
 from yt_dlp import YoutubeDL
 
-from yt_dlp.networking.common import Request
 from yt_dlp.networking.exceptions import RequestError, UnsupportedRequest
 from yt_dlp.utils import Popen
-from yt_dlp_plugins.extractor.getpot import GetPOTProvider, register_provider
+from yt_dlp_plugins.extractor.getpot import GetPOTProvider, register_provider, register_preference
+
+
+__version__ = '0.4.0'
 
 
 @register_provider
-class BgUtilPotProviderRH(GetPOTProvider):
-    _PROVIDER_NAME = 'BgUtilPot'
+class BgUtilScriptPotProviderRH(GetPOTProvider):
+    _PROVIDER_NAME = 'BgUtilScriptPot'
     _SUPPORTED_CLIENTS = ('web_creator', 'web', 'web_embedded', 'web_music')
+    VERSION = __version__
 
     def _validate_get_pot(self, client: str, ydl: YoutubeDL, visitor_data=None, data_sync_id=None, player_url=None, **kwargs):
+        script_path = ydl.get_info_extractor('Youtube')._configuration_arg(
+            'getpot_bgutil_script', [None], casesense=True)[0]
+        # TODO: Add default script path
```

Comment:
We can leave this for a seperate PR if u want

Reply:
done in 4b5d6ef13607e34519e05584a553685d65b5e938